[ERROR] Saving to DB

<!-- ## IMPORTANT -->
<!-- Issues are ONLY for reporting bugs and feature requests. For support, questions, queries and discussions use our slack channel. Slack Channel: https://mobsf.slack.com/ , Get Invitation: https://goo.gl/6cYU5a
-->
## EXPLANATION OF THE ISSUE
When i scanned a weixin apk, it report a bug,



The wechat version is 6.6.7 , download url
https://apkpure.com/cn/wechat/com.tencent.mm/download?from=details

The MobSF is the latest version of master branch



<!-- What happens, under which versions, under what conditions, when, and what were you expecting instead. -->

## STEPS TO REPRODUCE THE ISSUE

```
1. This is the first step
2. This is the second step
3. Further steps, etc.
```

## CONSOLE OUTPUT

```
[ERROR] Extracting Strings from APK (./StaticAnalyzer/views/android/strings.py, LINE 27 "for duo in rsrc.values[pkg][i]['string']:"): 'string'
 [ERROR] Saving to DB (./StaticAnalyzer/views/android/db_interaction.py, LINE 236 "static_db.save()"): StaticAnalyzer_staticanalyzerandroid.STRINGS may not be NULL
```

## CONTENTS OF LOG FILES

```
[INFO] Finished Code Analysis, Email and URL Extraction

[INFO] Generating Java and Smali Downloads
[INFO] Generating Downloads
[INFO] Zipping
[INFO] Zipping
[INFO] Extracting Strings from APK

[2018-08-16 03:25:58]
[ERROR] Extracting Strings from APK (./StaticAnalyzer/views/android/strings.py, LINE 27 "for duo in rsrc.values[pkg][i]['string']:"): 'string'

[INFO] Connecting to Database

[INFO] Saving to Database

[2018-08-16 03:25:58]
[ERROR] Saving to DB (./StaticAnalyzer/views/android/db_interaction.py, LINE 236 "static_db.save()"): StaticAnalyzer_staticanalyzerandroid.STRINGS may not be NULL
Thu Aug 16 03:25:58 2018 - SIGPIPE: writing to a closed pipe/socket/fd (probably the client disconnected) on request /StaticAnalyzer/?name=WeChat_v6.6.7_apkpure.com.apk&type=apk&checksum=4808f7184b9756fd8f436f3700eabf50 (ip 117.114.138.100) !!!
Thu Aug 16 03:25:58 2018 - uwsgi_response_write_body_do(): Broken pipe [core/writer.c line 341] during GET /StaticAnalyzer/?name=WeChat_v6.6.7_apkpure.com.apk&type=apk&checksum=4808f7184b9756fd8f436f3700eabf50 (117.114.138.100)
OSError: write error
```

```
If you have issues with API Fuzzer,
Paste the contents of logs/webproxy.log here
(NOT REQUIRED FOR FEATURE REQUEST/ENHANCEMENT)
```
<!--
BEFORE POSTING YOUR ISSUE/BUG
- These comments won't show up when you submit the issue.
- GitHub issues ARE NOT FOR SUPPORT, DISCUSSIONS AND QUESTIONS! 
- If you have questions, use our slack channel. Request invitation here https://goo.gl/6cYU5a.
- Installation problems with regards to requirements.txt or VirtualBox in not considered as a MobSF issue.
- Reproduce issue in the latest master and try to add as much detail as possible.
- If you're requesting a new feature/enhancement, explain why you'd like it to be added and it's importance.
- Search this repository (top of the page) for the issue and it has not been fixed or reported already.
- Once you open a bug, you should also provide additional information if requested. 
- Failure to do so in 25 days will result in closure of the bug without further communication.
-->

--- a/StaticAnalyzer/views/android/strings.py
+++ b/StaticAnalyzer/views/android/strings.py
@@ -24,7 +24,10 @@ def strings_jar(app_file, app_dir):
         pkg = rsrc.get_packages_names()[0]
         rsrc.get_strings_resources()
         for i in rsrc.values[pkg].keys():
-            for duo in rsrc.values[pkg][i]['string']:
+            string = rsrc.values[pkg][i].get('string')
+            if string is None:
+                return dat
+            for duo in string:
                 dat.append('"'+duo[0]+'" : "'+duo[1]+'"') 
         return dat
     except:

